test/test_HPE3ParClient_MockSSH.py

```diff
@@ -247,3 +248,44 @@ def test_strip_input_from_output(self):
                   'totals']
         result = ssh.HPE3PARSSHClient.strip_input_from_output(cmd, output)
         self.assertEqual(['out1', 'out2', 'out3'], result)
+
+    @mock.patch('hpe3parclient.client.HPE3ParClient.getPortNumber')
```

Comment:
By patching getPortNumber we are mocking it and in effect we are not executing getPortNumber flow at all.
What we need to patch is the SSH client which actually talks to the 3PAR array so that getPortNumber flow is executed but the SSH client call that actually connects to the array is mocked.